src/bun.js/webcore/response.classes.ts

```diff
@@ -153,7 +153,7 @@ export default [
       // This is *not* spec-compliant.
       name: {
         getter: "getName",
-        cache: true,
```

Comment:
We can have both `cache: true` and `setter: "setName"` and then the cached value will be kept alive but they can still overwrite it if necessary

Reply:
and then in `toJS` we can set the name on the Blob instance there

Reply:
That would probably simplify this and avoid worrying about leaking the name string

Reply:
We would need a strong ref for the JS `this` value so we can set/get the cached values, this would probably just be more complicated and we would still need to deinit one Strong ref, I can change to be a `bun.String` instead of a JSC.Strong, this would limit to be only a string but chrome dont limit the type on blob name to just string.
<img width="173" alt="image" src="https://github.com/oven-sh/bun/assets/6379399/1df2119b-1e16-40b8-a0c0-7733efceb159">

Reply:
I think it'd be okay to limit it to String until someone complains about that, and making it `configurable` so they can overwrite it with Object.defineProperty if they really want.

Reply:
ok will change to `bun.String` and make it `configurable`